doc/doxygen/doxygen-awesome-css/.github/workflows/publish.yaml

```diff
@@ -0,0 +1,29 @@
+name: publish
```

Comment:
are these workflows needed?

Reply:
I think we are pulling in too many unneeded files. e.g. github actions, docs, img folder need to go. 
maybe also include/MyLibrary (did not check)